lib/generators/module/module_generator.rb

```diff
@@ -51,15 +51,17 @@ def create_additional_files
   def update_and_install
     # spec helper add group
     insert_into_file 'spec/spec_helper.rb', "\tadd_group '#{file_name.camelize}',
-      'modules/#{file_name}/'\n", after: "# Modules\n"
+                     'modules/#{file_name}/'\n", after: "# Modules\n"
 
     # simplecov add group
     insert_into_file 'spec/simplecov_helper.rb', "\tadd_group '#{file_name.camelize}',
-      'modules/#{file_name}/'\n", after: "# Modules\n"
+                     'modules/#{file_name}/'\n", after: "# Modules\n"
 
     # insert into main app gemfile
     insert_into_file 'Gemfile', "\tgem '#{file_name}'\n", after: "path 'modules' do\n"
-    route "mount #{file_name.camelize}::Engine, at: '/#{file_name}'"
+
+    insert_into_file 'config/routes.rb',
+                     "\tmount #{file_name.camelize}::Engine, at: '/#{file_name}'\n", after: "# Modules\n"
```

Comment:
`\t` is causing rubocop to be unhappy.

Reply:
Looks like checks are passing ✅

Reply:
I mean when you run the generator.

Reply:
```
git checkout master
rails generate module foo
rubocop
# ... 62 offenses detected
```

```
git checkout tweak-module-generator-file-insert
rails generate module foo
rubocop
# ... 76 offenses detected
```

Reply:
I'd be fine with a ticket that addresses this so we can merge this PR.

Reply:
Ticket created [here](https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/18726)